### ISSUE TITLE
OAuth support for python2

### ISSUE THREAD

**Issue by michael-lazar** [michael-lazar/rtv#137]
Program crashes upon startup using python 2.7.6

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/mlazar/Misc/rtv/rtv/__main__.py", line 187, in <module>
    sys.exit(main())
  File "/home/mlazar/Misc/rtv/rtv/__main__.py", line 125, in main
    local_oauth_config = load_oauth_config()
  File "/home/mlazar/Misc/rtv/rtv/__main__.py", line 82, in load_oauth_config
    config['oauth'] = {'auto_login': False}
AttributeError: ConfigParser instance has no attribute '__setitem__'
```

**Comment by TheoPib** [comment]
This is because ConfigParser is an instance instead of an object in Python 2. We can use ConfigParser methods instead.

**Comment by TheoPib** [comment]
Are you able to run rtv on Python 2 ? I tried, but there are some compatibility issues. For example :

```
from . import config
ValueError: Attempted relative import in non-package
```

**Comment by michael-lazar** [comment]
Yea, how are you running it? What I do is *cd* into the root of the project and run it as a module.
```
ubuntu:~/projects/rtv$ ls
CHANGELOG.rst	  LICENSE      __pycache__  rtv      setup.cfg	tests
CONTRIBUTORS.rst  MANIFEST.in  README.rst   rtv.log  setup.py	version.py

ubuntu:~/Misc/rtv$ python2 -m rtv
Connecting...
```

**Comment by TheoPib** [comment]
OK, I ran it with ```python2 rtv```. Maybe this should appear in the README then?

**Comment by michael-lazar** [comment]
Yea that's probably a good idea :smile:

**Comment by michael-lazar** [comment]
Merged in #141
